packages/esm-tb-app/src/views/dashboard/summary-tiles/tb-summary-tiles.component.tsx

```diff
@@ -1,11 +1,24 @@
 import { OHRIProgrammeSummaryTiles } from '@ohri/openmrs-esm-ohri-commons-lib';
 import React, { useEffect, useMemo, useState } from 'react';
 import { useTranslation } from 'react-i18next';
+import { getActiveDsCasesCount, getActiveDrCasesCount, } from '../../../api/api';
 
 function TbSummaryTiles({ launchWorkSpace }) {
   const { t } = useTranslation();
-  const [activeDSClientsCount, setActiveDSClientsCount] = useState(35);
-  const [activeDRClientsCount, setActiveDRClientsCount] = useState(5);
+  const [activeDSClientsCount, setActiveDSClientsCount] = useState(null);
+  const [activeDRClientsCount, setActiveDRClientsCount] = useState(null);
+
+  useEffect(() => {
+    getActiveDsCasesCount().then((count) => {
+      setActiveDSClientsCount(count);
+    });
+  }, []);
+
+  useEffect(() => {
+    getActiveDrCasesCount().then((count) => {
+      setActiveDRClientsCount(count);
+    });
+  }, []);
```

Comment:
The api calls can be moved here and using the hook would not necessitate a useEffect